app/controllers/v0/sessions_controller.rb

```diff
@@ -112,5 +138,11 @@ def build_logout_errors(logout_response, logout_request, session, user)
     def saml_options
       Settings.review_instance_slug.blank? ? {} : { RelayState: Settings.review_instance_slug }
     end
+
+    # NOTE: this is going to break review instances. How can we make RelayState dual use?
+    def fetch_url(authn_context = nil)
```

Comment:
I find this name confusing, I'd call it `build_url` - fetch sounds like it's getting a remote resource, this is just constructing the Saml AuthnRequest URL from settings and parameters.